### PR TITLE
Model migrations with Alembic

### PR DESCRIPTION
**Pull request by MillarayVVS**
# Summary

Adds Alembic to use for migrations and modifies dataset model deleting task_name.

## Type of change

- Back end new feature.
- Documentation.

## Changes

- Install Alembic for migration management.
- Adds How to make migrations with Alembic to the README file
- Remove task_name from dataset model with migrations
- Modify tests and validations for dataset model.

## How to Test

- Run ```pip install -r requirements.txt```
- Run ```alembic upgrade head``` to apply migrations scripts
- Run ```alembic current``` to check the current migration status

### CODE REVIEW

File: tests/back/api/test_datasets.py [hunk 1]
```diff
@@ -79,10 +76,10 @@ def test_create_csv_dataset(client: TestClient):
 
 def test_get_all_datasets(client: TestClient):
     response = client.get("/api/v1/dataset/")
+    print(response)
```
Comment (cristian-tamblay) [inline tests/back/api/test_datasets.py]: Please avoid merging code with prints

File: tests/back/dataloaders/dashaidataset/test/data-00000-of-00001.arrow
Comment (cristian-tamblay) [inline tests/back/dataloaders/dashaidataset/test/data-00000-of-00001.arrow:1]: What are these files?

File: alembic/README
Comment (cristian-tamblay) [inline alembic/README:1]: Is this file needed?

File: README.rst [hunk 2]
```diff
@@ -187,6 +187,22 @@ There are two ways to run DashAI:
...
+
+.. code:: bash
+
+    $ alembic revision --autogenerate -m "Describe the changes you did here"
+
+2. Apply migrations:
+
+.. code:: bash
+
+    $ alembic upgrade head
```
Comment (cristian-tamblay) [inline README.rst]: Do i run these commands from the root or inside the alembic folder?

**Review by cristian-tamblay — CHANGES_REQUESTED** [review]
Seems good, i left some minor changes and we should be good to merge. Also, why merge it to data-studio instead of staging?

File: tests/back/api/test_datasets.py (hunk 1 above)
Comment (MillarayVVS) [reply tests/back/api/test_datasets.py]: Resolved in recent commit

File: README.rst (hunk 2 above)
Comment (MillarayVVS) [reply README.rst]: You should run these commands from the root, I added a new instruction in the README file on the recent commit.

**Comment by cristian-tamblay** [comment]
Please Resolve conflicts so we can merge this! You can follow the `Use the command line to resolve conflicts` on the box below

**Pull request closed without merging**